shell.nix

```diff
@@ -0,0 +1,23 @@
+{ pkgs ? import <nixpkgs> {} }:
+
+/*
+nix shell for development usage
+1. run `nix-shell`
+2. run `make run` (or any other make scripts)
+Done!
+*/
+
+(pkgs.buildFHSEnv {
```

Comment:
Why didn't you use mkShell?

Reply:
Didn't let me run the node nvm stuff from the makefile otherwise! I'm on NixOS 24.11 but in case this was a my-config-issue then we can switch it

Reply:
Do you still have the error?

Reply:
Why are you using nvm? NixOS has [nodejs_23](https://search.nixos.org/packages?channel=24.11&from=0&size=50&sort=relevance&type=packages&query=npm) and we should probably use that because then we avoid issues with certs and don't need any kind of patching.